Algorithmia/AlgorithmiaClient.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  AlgorithmiaClient.swift
+//  AlgorithmiaSwiftDemo
+//
+//  Created by Erik Ilyin on 11/1/16.
+//  Copyright © 2016 algorithmia. All rights reserved.
+//
+
+import Foundation
+
+class AlgorithmiaClient {
+    var apiClient:AlgoAPIClient
+    init() {
+        self.apiClient = AlgoAPIClient(auth: nil)
```

Comment:
Default client should use the environment variable `ALGORITHMIA_API_KEY` as the API key if it is available. If it's not, then `nil` is fine.